<?xml version="1.0" encoding="utf-8"?>
<!--
This URDF was automatically created by SolidWorks to URDF Exporter! Originally created by Stephen Brawner (brawner@gmail.com)
Commit Version: 1.6.0-4-g7f85cfe  Build Version: 1.6.7995.38578
For more information, please see http://wiki.ros.org/sw_urdf_exporter
-->
<robot name="pai02_1_12_240807">
	<link name="base_link">
		<inertial>
			<origin xyz="-0.00011969 0.00431563 0.07060951" rpy="0 0 0" />
			<mass value="2.47789918" />
			<inertia ixx="0.0079505" ixy="2.457E-05"  ixz="2.836E-05" 
									 iyy="0.00983126" iyz="-1.421E-05" 
									    			  izz="0.00758548" />
		</inertial>
		<visual>
			<origin xyz="0 0 0" rpy="0 0 0" />
			<geometry>
				<mesh filename="package://pai02_1_12_240807/meshes/base_link.STL" />
			</geometry>
			<material name="">
				<color rgba="0.89804 0.91765 0.92941 1" />
			</material>
		</visual>
		<collision>
			<origin xyz="0 0 0" rpy="0 0 0" />
			<geometry>
				<mesh filename="package://pai02_1_12_240807/meshes/base_link.STL" />
			</geometry>
		</collision>
	</link>
	<link name="r_hip_yaw_link">
		<inertial>
			<origin xyz="-0.00012202 -0.05213112 -0.05273544" rpy="0 0 0" />
			<mass value="0.34953789" />
			<inertia ixx="0.00025249" ixy="-4.5E-07" ixz="5.3E-07" iyy="0.00022813" iyz="-5.884E-05" izz="0.00016829" />
		</inertial>
		<visual>
			<origin xyz="0 0 0" rpy="0 0 0" />
			<geometry>
				<mesh filename="package://pai02_1_12_240807/meshes/r_hip_yaw_link.STL" />
			</geometry>
			<material name="">
				<color rgba="0.89804 0.91765 0.92941 1" />
			</material>
		</visual>
		<collision>
			<origin xyz="0 0 0" rpy="0 0 0" />
			<geometry>
				<mesh filename="package://pai02_1_12_240807/meshes/r_hip_yaw_link.STL" />
			</geometry>
		</collision>
	</link>
	<joint name="r_hip_yaw_joint" type="revolute">
		<origin xyz="0.075 0 0" rpy="0 0 0" />
		<parent link="base_link" />
		<child link="r_hip_yaw_link" />
		<axis xyz="0 0 1" />
		<limit lower="-3.14" upper="3.14" effort="21" velocity="21" />
	</joint>
	<link name="r_hip_roll_link">
		<inertial>
			<origin xyz="-0.00022682 -0.00214086 -0.00016674" rpy="0 0 0" />
			<mass value="0.67430799" />
			<inertia ixx="0.00031647" ixy="4.2E-07" ixz="3.61E-06" iyy="0.00098969" iyz="4.2E-07" izz="0.00101632" />
		</inertial>
		<visual>
			<origin xyz="0 0 0" rpy="0 0 0" />
			<geometry>
				<mesh filename="package://pai02_1_12_240807/meshes/r_hip_roll_link.STL" />
			</geometry>
			<material name="">
				<color rgba="0.89804 0.91765 0.92941 1" />
			</material>
		</visual>
		<collision>
			<origin xyz="0 0 0" rpy="0 0 0" />
			<geometry>
				<mesh filename="package://pai02_1_12_240807/meshes/r_hip_roll_link.STL" />
			</geometry>
		</collision>
	</link>
	<joint name="r_hip_roll_joint" type="revolute">
		<origin xyz="0 0 -0.059" rpy="0 0 0" />
		<parent link="r_hip_yaw_link" />
		<child link="r_hip_roll_link" />
		<axis xyz="0 1 0" />
		<limit lower="-3.14" upper="3.14" effort="21" velocity="21" />
	</joint>
	<link name="r_thigh_link">
		<inertial>
			<origin xyz="0.00051013 0.0023326 -0.096411" rpy="0 0 0" />
			<mass value="0.19993" />
			<inertia ixx="0.00064072" ixy="2.1E-07" ixz="-9.29E-06" iyy="0.00063292" iyz="7.5E-06" izz="3.516E-05" />
		</inertial>
		<visual>
			<origin xyz="0 0 0" rpy="0 0 0" />
			<geometry>
				<mesh filename="package://pai02_1_12_240807/meshes/r_thigh_link.STL" />
			</geometry>
			<material name="">
				<color rgba="0.89804 0.91765 0.92941 1" />
			</material>
		</visual>
		<collision>
			<origin xyz="0 0 0" rpy="0 0 0" />
			<geometry>
				<mesh filename="package://pai02_1_12_240807/meshes/r_thigh_link.STL" />
			</geometry>
		</collision>
	</link>
	<joint name="r_thigh_joint" type="revolute">
		<origin xyz="0 0 0" rpy="0 0 0" />
		<parent link="r_hip_roll_link" />
		<child link="r_thigh_link" />
		<axis xyz="1 0 0" />
		<limit lower="-3.14" upper="3.14" effort="21" velocity="21" />
	</joint>
	<link name="r_calf_link">
		<inertial>
			<origin xyz="6.957E-05 -0.00092506 -0.09475741" rpy="0 0 0" />
			<mass value="0.54984799" />
			<inertia ixx="0.00073284" ixy="8.7E-07" ixz="-6.748E-05" iyy="0.00073948" iyz="-3.988E-05" izz="0.00027251" />
		</inertial>
		<visual>
			<origin xyz="0 0 0" rpy="0 0 0" />
			<geometry>
				<mesh filename="package://pai02_1_12_240807/meshes/r_calf_link.STL" />
			</geometry>
			<material name="">
				<color rgba="0.89804 0.91765 0.92941 1" />
			</material>
		</visual>
		<collision>
			<origin xyz="0 0 0" rpy="0 0 0" />
			<geometry>
				<mesh filename="package://pai02_1_12_240807/meshes/r_calf_link.STL" />
			</geometry>
		</collision>
	</link>
	<joint name="r_calf_joint" type="revolute">
		<origin xyz="0 0 -0.15" rpy="0 0 0" />
		<parent link="r_thigh_link" />
		<child link="r_calf_link" />
		<axis xyz="1 0 0" />
		<limit lower="-3.14" upper="3.14" effort="21" velocity="21" />
	</joint>
	<link name="r_ankle_pitch_link">
		<inertial>
			<origin xyz="0 0 0" rpy="0 0 0" />
			<mass value="0.01158437" />
			<inertia ixx="1.12E-06" ixy="0" ixz="0" iyy="4.5E-07" iyz="0" izz="1.21E-06" />
		</inertial>
		<visual>
			<origin xyz="0 0 0" rpy="0 0 0" />
			<geometry>
				<mesh filename="package://pai02_1_12_240807/meshes/r_ankle_pitch_link.STL" />
			</geometry>
			<material name="">
				<color rgba="0.65098 0.61961 0.58824 1" />
			</material>
		</visual>
		<collision>
			<origin xyz="0 0 0" rpy="0 0 0" />
			<geometry>
				<mesh filename="package://pai02_1_12_240807/meshes/r_ankle_pitch_link.STL" />
			</geometry>
		</collision>
	</link>
	<joint name="r_ankle_pitch_joint" type="revolute">
		<origin xyz="0 0 -0.17545" rpy="0 0 0" />
		<parent link="r_calf_link" />
		<child link="r_ankle_pitch_link" />
		<axis xyz="1 0 0" />
		<limit lower="-3.14" upper="3.14" effort="21" velocity="21" />
	</joint>
	<link name="r_ankle_roll_link">
		<inertial>
			<origin xyz="2.06E-06 0.01663117 -0.0211042" rpy="0 0 0" />
			<mass value="0.26606111" />
			<inertia ixx="0.00039542" ixy="1E-08" ixz="-2E-08" iyy="0.00010328" iyz="2.208E-05" izz="0.00045941" />
		</inertial>
		<visual>
			<origin xyz="0 0 0" rpy="0 0 0" />
			<geometry>
				<mesh filename="package://pai02_1_12_240807/meshes/r_ankle_roll_link.STL" />
			</geometry>
			<material name="">
				<color rgba="0.89804 0.91765 0.92941 1" />
			</material>
		</visual>
		<collision>
			<origin xyz="0 0 0" rpy="0 0 0" />
			<geometry>
				<mesh filename="package://pai02_1_12_240807/meshes/r_ankle_roll_link.STL" />
			</geometry>
		</collision>
	</link>
	<joint name="r_ankle_roll_joint" type="revolute">
		<origin xyz="0 0 0" rpy="0 0 0" />
		<parent link="r_ankle_pitch_link" />
		<child link="r_ankle_roll_link" />
		<axis xyz="0 1 0" />
		<limit lower="-3.14" upper="3.14" effort="21" velocity="21" />
	</joint>
	<link name="l_hip_yaw_link">
		<inertial>
			<origin xyz="6.958E-05 -0.05287777 -0.05288161" rpy="0 0 0" />
			<mass value="0.34953789" />
			<inertia ixx="0.00025057" ixy="2.6E-07" ixz="-3.9E-07" iyy="0.00023311" iyz="-5.987E-05" izz="0.00016652" />
		</inertial>
		<visual>
			<origin xyz="0 0 0" rpy="0 0 0" />
			<geometry>
				<mesh filename="package://pai02_1_12_240807/meshes/l_hip_yaw_link.STL" />
			</geometry>
			<material name="">
				<color rgba="0.89804 0.91765 0.92941 1" />
			</material>
		</visual>
		<collision>
			<origin xyz="0 0 0" rpy="0 0 0" />
			<geometry>
				<mesh filename="package://pai02_1_12_240807/meshes/l_hip_yaw_link.STL" />
			</geometry>
		</collision>
	</link>
	<joint name="l_hip_yaw_joint" type="revolute">
		<origin xyz="-0.075 0 0" rpy="0 0 0" />
		<parent link="base_link" />
		<child link="l_hip_yaw_link" />
		<axis xyz="0 0 1" />
		<limit lower="-3.14" upper="3.14" effort="21" velocity="21" />
	</joint>
	<link name="l_hip_roll_link">
		<inertial>
			<origin xyz="0.00022682 -0.00198932 -0.00016674" rpy="0 0 0" />
			<mass value="0.67430799" />
			<inertia ixx="0.00032559" ixy="-3.9E-07" ixz="-2.13E-06" iyy="0.00101903" iyz="4E-07" izz="0.00104452" />
		</inertial>
		<visual>
			<origin xyz="0 0 0" rpy="0 0 0" />
			<geometry>
				<mesh filename="package://pai02_1_12_240807/meshes/l_hip_roll_link.STL" />
			</geometry>
			<material name="">
				<color rgba="1 1 1 1" />
			</material>
		</visual>
		<collision>
			<origin xyz="0 0 0" rpy="0 0 0" />
			<geometry>
				<mesh filename="package://pai02_1_12_240807/meshes/l_hip_roll_link.STL" />
			</geometry>
		</collision>
	</link>
	<joint name="l_hip_roll_joint" type="revolute">
		<origin xyz="0 0 -0.059" rpy="0 0 0" />
		<parent link="l_hip_yaw_link" />
		<child link="l_hip_roll_link" />
		<axis xyz="0 1 0" />
		<limit lower="-3.14" upper="3.14" effort="21" velocity="21" />
	</joint>
	<link name="l_thigh_link">
		<inertial>
			<origin xyz="-0.00034834 0.0023012 -0.095112" rpy="0 0 0" />
			<mass value="0.20265649" />
			<inertia ixx="0.00066685" ixy="-1.3E-07" ixz="6.17E-06" iyy="0.00065889" iyz="8.11E-06" izz="3.611E-05" />
		</inertial>
		<visual>
			<origin xyz="0 0 0" rpy="0 0 0" />
			<geometry>
				<mesh filename="package://pai02_1_12_240807/meshes/l_thigh_link.STL" />
			</geometry>
			<material name="">
				<color rgba="0.89804 0.91765 0.92941 1" />
			</material>
		</visual>
		<collision>
			<origin xyz="0 0 0" rpy="0 0 0" />
			<geometry>
				<mesh filename="package://pai02_1_12_240807/meshes/l_thigh_link.STL" />
			</geometry>
		</collision>
	</link>
	<joint name="l_thigh_joint" type="revolute">
		<origin xyz="0 0 0" rpy="0 0 0" />
		<parent link="l_hip_roll_link" />
		<child link="l_thigh_link" />
		<axis xyz="1 0 0" />
		<limit lower="-3.14" upper="3.14" effort="21" velocity="21" />
	</joint>
	<link name="l_calf_link">
		<inertial>
			<origin xyz="-7.122E-05 -0.00115019 -0.09475743" rpy="0 0 0" />
			<mass value="0.54984829" />
			<inertia ixx="0.00069528" ixy="-8.7E-07" ixz="6.083E-05" iyy="0.00070372" iyz="-3.904E-05" izz="0.00023616" />
		</inertial>
		<visual>
			<origin xyz="0 0 0" rpy="0 0 0" />
			<geometry>
				<mesh filename="package://pai02_1_12_240807/meshes/l_calf_link.STL" />
			</geometry>
			<material name="">
				<color rgba="0.89804 0.91765 0.92941 1" />
			</material>
		</visual>
		<collision>
			<origin xyz="0 0 0" rpy="0 0 0" />
			<geometry>
				<mesh filename="package://pai02_1_12_240807/meshes/l_calf_link.STL" />
			</geometry>
		</collision>
	</link>
	<joint name="l_calf_joint" type="revolute">
		<origin xyz="0 0 -0.15" rpy="0 0 0" />
		<parent link="l_thigh_link" />
		<child link="l_calf_link" />
		<axis xyz="1 0 0" />
		<limit lower="-3.14" upper="3.14" effort="21" velocity="21" />
	</joint>
	<link name="l_ankle_pitch_link">
		<inertial>
			<origin xyz="0 0 0" rpy="0 0 0" />
			<mass value="0.01158437" />
			<inertia ixx="1.12E-06" ixy="0" ixz="0" iyy="4.5E-07" iyz="0" izz="1.21E-06" />
		</inertial>
		<visual>
			<origin xyz="0 0 0" rpy="0 0 0" />
			<geometry>
				<mesh filename="package://pai02_1_12_240807/meshes/l_ankle_pitch_link.STL" />
			</geometry>
			<material name="">
				<color rgba="0.89804 0.91765 0.92941 1" />
			</material>
		</visual>
		<collision>
			<origin xyz="0 0 0" rpy="0 0 0" />
			<geometry>
				<mesh filename="package://pai02_1_12_240807/meshes/l_ankle_pitch_link.STL" />
			</geometry>
		</collision>
	</link>
	<joint name="l_ankle_pitch_joint" type="revolute">
		<origin xyz="0 0 -0.17545" rpy="0 0 0" />
		<parent link="l_calf_link" />
		<child link="l_ankle_pitch_link" />
		<axis xyz="1 0 0" />
		<limit lower="-3.14" upper="3.14" effort="21" velocity="21" />
	</joint>
	<link name="l_ankle_roll_link">
		<inertial>
			<origin xyz="0 0.01700127 -0.0213352" rpy="0 0 0" />
			<mass value="0.26318612" />
			<inertia ixx="0.00039075" ixy="0" ixz="2E-08" iyy="0.00010189" iyz="2E-05" izz="0.00045603" />
		</inertial>
		<visual>
			<origin xyz="0 0 0" rpy="0 0 0" />
			<geometry>
				<mesh filename="package://pai02_1_12_240807/meshes/l_ankle_roll_link.STL" />
			</geometry>
			<material name="">
				<color rgba="0.89804 0.91765 0.92941 1" />
			</material>
		</visual>
		<collision>
			<origin xyz="0 0 0" rpy="0 0 0" />
			<geometry>
				<mesh filename="package://pai02_1_12_240807/meshes/l_ankle_roll_link.STL" />
			</geometry>
		</collision>
	</link>
	<joint name="l_ankle_roll_joint" type="revolute">
		<origin xyz="0 0 0" rpy="0 0 0" />
		<parent link="l_ankle_pitch_link" />
		<child link="l_ankle_roll_link" />
		<axis xyz="0 1 0" />
		<limit lower="-3.14" upper="3.14" effort="21" velocity="21" />
	</joint>
</robot>
